ruby/ql/src/queries/security/cwe-079/examples/unsafe-html-construction.rb

```diff
@@ -0,0 +1,6 @@
+class UsersController < ActionController::Base
+  # BAD - create a user description, where the name is not escaped
+  def create_user_description (name)
+    "<h2>#{name}</h2>".html_safe
```

Comment:
```suggestion
    "<b>#{name}</b>".html_safe
```

to match the documentation for this example